### PR TITLE
FIX : TT-1596 : Document new way of using headers

### PR DESCRIPTION
**Pull request by jay-deshmukh**
- This PR makes a fix to document new way of using headers when debugging graphql apis with graphiql IDE on dashboard.

### CODE REVIEW

**Review by marksou — APPROVED** [review]

File: tyk-docs/content/graphql/graphql-playground.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The GraphiQL try-out playground comes with a series of features by default, whic
 
   **Headers**
 
-  Debugging a GraphQL API might require additional headers to be passed to the requests while playing with the GraphiQL interface. (i.e. `Authorization` header in case of Authentication Token protection over the API);
+  Debugging a GraphQL API might require additional headers to be passed to the requests while playing with the GraphiQL interface. (i.e. `Authorization` header in case of Authentication Token protection over the API), this can be done using the dedicated headers tab in Graphiql IDE
```
Comment (letzya) [inline tyk-docs/content/graphql/graphql-playground.md]: @marksou can you please check the commas and full stops are placed correctly? thanks

Comment (marksou) [reply tyk-docs/content/graphql/graphql-playground.md]: Done now.

Comment (jay-deshmukh) [reply tyk-docs/content/graphql/graphql-playground.md]: Thanks @letzya and @marksou

**Comment by letzya** [comment]
@marksou can we maybe add a screenshot of it?

**Review by letzya — CHANGES_REQUESTED** [review]
A screenshot can help

**Pull request merged**